meta/documents/user_guide/de/plugin_installation.md

```diff
@@ -0,0 +1,92 @@
+
+3. Konfigurieren Sie die Plugin-Sets, die von Plentymarkets verwendet werden sollen
+    * Das Feld `Verknüpfte Mandanten` für das Plugin-Set sollte die Namen aller Plentymarkets Mandanten enthalten, für die die Wayfair-Plugin-Funktionalität gewünscht wird.
+
+        ![linked clients](../../../images/de/plugin_sets_linked_clients.png)
+
+    * Verwenden Sie optional die Schaltflächen `Mandanten und Plugin-Sets verknüpfen`, um die Verknüpfungen zwischen Mandanten und Plugin-Sets zu ändern.
+        ![link plugin sets button](../../../images/de/installation/button_link_plugin_sets.png)
+
+4. Klicken Sie auf die Schaltfläche `Plugin-Set bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das Plugin-Set, in dem das Wayfair-Plugin installiert werden soll.
+
+5. Klicken Sie auf die Schaltfläche `+ Plugin hinzufügen`
+    ![add plugin button](../../../images/de/installation/button_add_plugin.png)
+
+6. `Wayfair` sollte in der Liste der Plugins erscheinen
+
+    * Wenn es viele Plugins gibt, geben Sie "Wayfair" in das Feld `Suchen` ein
+
+    ![search_for_wayfair_plugin](../../../images/de/installation/search_for_wayfair_plugin.png)
+
+7. Klicken Sie in der Liste auf den Eintrag `Wayfair`. Das Wayfair-Logo sollte mit einer Schaltfläche `Installieren` angezeigt werden. (![marketplace icon](../../../images/common/icon_marketplace.png))
+
+    * Wenn Sie das Wayfair-Plugin zuvor über den Kanal `git` installiert haben, gibt es möglicherweise mehrere Einträge für `Wayfair`. Bitte wählen Sie den Eintrag mit der Quelle `Marktplatz`.
+
+8. Überprüfen Sie die Informationen auf der Seite.
+
+    ![wayfair plugin info](../../../images/de/installation/marketplace_wayfair.png)
+
+9. Wählen Sie im Dropdown-Menü "Version auswählen" die Option [die neueste Plugin-Version](https://github.com/wayfair-contribs/plentymarkets-plugin/releases), falls diese noch nicht ausgewählt ist.
+
+10. Klicken Sie oben auf der Seite auf die Schaltfläche `Installieren` ![install button](../../../images/de/installation/button_install.png). Sie werden zu den Details des Plugin-Sets weitergeleitet.
+
+11. Stellen Sie sicher, dass auf der Seite eine Zeile mit dem Namen `Wayfair` und der installierten Version als gewünschte Version vorhanden ist.
+
+12. Klicken Sie in der Spalte `Aktiv` für die Zeile `Wayfair` auf den Schalter, damit er sich in die aktivierte Position auf der rechten Seite bewegt. **Wenn das Wayfair-Plugin nicht aktiviert wird, sind seine Funktionen nicht verfügbar**.
+
+13. Klicken Sie oben auf der Seite auf die Schaltfläche `Plugin-Set bereitstellen` (wird als Schaltfläche "Speichern" angezeigt). Während der Installation des Plugins wird ein Fortschrittsbalken angezeigt.
+
+    ![wayfair plugin not yet deployed](../../../images/de/installation/wayfair_plugin_not_yet_deployed.png)
+
+14. Vergewissern Sie sich, dass die Spalte `Bereitgestellt` für die Zeile `Wayfair` jetzt ausgefüllt ist und den Wert in der Spalte `Installiert` widerspiegelt.

```

Comment:
Translation does not match the steps/instructions? This seems like it should be for Step 13

Reply:
```suggestion
14. Klicken Sie oben auf der Seite auf die Schaltfläche Plugin-Set bereitstellen (wird als Schaltfläche "Speichern" angezeigt). 
```
Looks like the instructions for Step 14 were pasted in Step 13

Reply:
the translation is fine, screenshot for step 13 and 14 were interchanged